src/layouts/app/translations-mixin.ts

```diff
@@ -43,15 +47,15 @@ export default (superClass: Constructor<LitElement & HassBaseEl>) =>
         return;
       }
 
-      const language = hass.selectedLanguage || hass.language;
+      const language = hass.language;
 
       const { resources } = await hass.callWS({
```

Comment:
One day we'll move this to `src/data/frontend.ts`

Reply:
Done